manticore/ethereum.py

```diff
@@ -1549,13 +1549,13 @@ def global_coverage(self, account_address):
         world = None
         for state_id in self.all_state_ids:
             world = self.get_world(state_id)
-            if account_address in world.storage:
+            if account_address in world.contract_accounts:
                 break
 
         with self.locked_context('runtime_coverage') as coverage:
             seen = coverage
-        #runtime_bytecode = world.storage[account_address]['code']
-        runtime_bytecode = self.get_metadata(account_address).runtime_bytecode
+        runtime_bytecode = world.get_code(account_address)[:-44]
```

Comment:
let's not use the magic number -44 here, either add a comment or a variable or something?